sentry/src/main/java/io/sentry/Breadcrumb.java

```diff
@@ -59,6 +60,25 @@ public Breadcrumb(final @NotNull Date timestamp) {
     this.level = breadcrumb.level;
   }
 
+  /**
+   * Creates breadcrumb from a map.
+   *
+   * @param map - The breadcrumb data as map
+   * @param options - the sentry options
+   * @return the breadcrumb
+   */
+  public static @Nullable Breadcrumb fromMap(
+      @NotNull Map<String, Object> map, @NotNull SentryOptions options) {
+    try {
+      String json = options.getSerializer().serialize(map);
```

Comment:
tbqh, I'm unsure about this approach with ser/deser roundtrip. Is there a specific reason for that, other than less code? I'd prefer just a simple approach with iterating over key-values and having if-checks for each key to construct a breadcrumb.

With the current approach, we potentially also need to call this from a background thread on the consumer's side

Reply:
Yeah, the reason was to leverage existing (tested) code to do the deser. It is runtime overhead though, i will change as suggested.